lib/remap-lang/src/parser.rs

```diff
@@ -753,8 +765,8 @@ mod tests {
             (
                 ".foo bar",
                 vec![
-                    " 1:6\n",
-                    "= expected assignment, if-statement, query, operator, or block",
+                    " 1:9\n",
```

Comment:
What do these numbers represent? It wasn't clear to me which made it also unclear to me why they are changing as the result of this PR :smile:

Reply:
line and column numbers. Its an awkward way to test, but I copied these over from what we already had. I plan on updating all these tests to use a macro at some point, to make it more clear what's happening here.

Reply:
Aha I see it :+1: